index.js

```diff
@@ -1,59 +1,75 @@
 const arr = JSON.parse(localStorage.getItem('array')) || [];
 
```

Comment:
## Project Requirement 📝
![image](https://user-images.githubusercontent.com/93039359/227170081-20582bea-2975-4d70-8473-06172234bd1d.png)


- [ ] It would be great if you have this array inside your Books class to follow the project requirements 😊 When you create an object from a class then that object should have an array of items that could be updated in the future 👍 Kindly use this array inside your Class 👍

**Hint: You can have this array in your constructor method 👍**